src/platform/utilities/tests/api/index.unit.spec.js

```diff
@@ -73,6 +73,25 @@ describe('apiRequest', () => {
       expect(error).to.deep.equal(JSON.parse(expected.response.body));
     });
   });
+
+  it('should return JSON on (status: 403)', async () => {
```

Comment:
@asg5704 I think this is basically what you were asking for, let me know if this needs any alterations